m4/pdns_with_gnutls.m4

```diff
@@ -18,7 +18,7 @@ AC_DEFUN([PDNS_WITH_GNUTLS], [
         save_LIBS=$LIBS
         CFLAGS="$GNUTLS_CFLAGS $CFLAGS"
         LIBS="$GNUTLS_LIBS $LIBS"
-        AC_CHECK_FUNCS([gnutls_memset])
+        AC_CHECK_FUNCS([gnutls_memset, gnutls_session_set_verify_cert])
```

Comment:
We don't usually use commas between the functions, does that work?

Reply:
it does work, but fixing to conform anyway